### ISSUE TITLE
Non-free test data

### ISSUE THREAD

**Issue by jayvdb** [HazyResearch/pdftotree#116]
While looking further into #114, I found that some of the docs in `tests/input` are copyrighted and non-free, and thus should not be included in GitHub unless there is some explicit permission from the rights holders.  As one of them is https://www.sciencedirect.com/science/article/abs/pii/S1464343X10000865 , it is quite likely that rights holder has not, and will not, give free-libre rights for its redistribution.

I suggest finding alternative PDFs for testing, and delete the copyrighted PDFs before another release is cut.

**Comment by lukehsiao** [comment]
You're absolutely right. It should be pretty easy to use LaTeX and some lorem ipsum text to generate our own test documents.
